### PR TITLE
Update `file` dependency.

### PR DESCRIPTION
**Pull request by bernaferrari**
Needed for https://github.com/flutter/flutter/pull/123260.

### CODE REVIEW

**Comment by elliette** [comment]
> `dart_code_metrics` is clashing, but that package is deprecated. Should we remove it?

Yes, find to remove `dart_code_metrics`. Thanks!

File: dwds/analysis_options.yaml [hunk 1]
```diff
@@ -23,66 +23,3 @@ linter:
     - unawaited_futures
     - unnecessary_lambdas
 
-# TODO(https://github.com/dart-lang/webdev/issues/2053): Enable commented-out rules with fixes.
```
Comment (elliette) [inline dwds/analysis_options.yaml:26]: Oh we still want the rules, we just don't need the pub dependency (dart code metrics gets run as a github workflow, and will notify contributor of any changes that are needed on the PR)

Comment (bernaferrari) [reply dwds/analysis_options.yaml:26]: But if I remove from the pub, will it still be run in the workflow?

Comment (elliette) [reply dwds/analysis_options.yaml:26]: Yes we actually never needed it in the pubspec (we are downloading it in the github workflow). I forgot to remove it from the pubspec a while back 🤦‍♀️

Comment (bernaferrari) [reply dwds/analysis_options.yaml]: Ok, here it goes again.

**Review by elliette — APPROVED** [review]
Thank you!

**Pull request merged**